include/psa/crypto.h

```diff
@@ -2585,8 +2590,8 @@ psa_status_t psa_aead_update_ad(psa_aead_operation_t *operation,
  * \retval #PSA_SUCCESS
  *         Success.
  * \retval #PSA_ERROR_BAD_STATE
- *         The operation state is not valid (not set up, nonce not set
- *         or already completed).
+ *         The operation state is not valid (it must be active and ready to
```

Comment:
Saying that this function fails when the operation is not ready for this function to be called doesn't explain much.

> (it must be active, have a nonce set, and have lengths set if required by the algorithm)